Embedded/MaxMix/MaxMix.ino

```diff
@@ -178,8 +185,12 @@ void loop()
     UpdateLighting();
   }
 
-  Display::UpdateTimers(now - last);
-  isDirty = false;
+  // Reset / Disconnect if no serial activity.
+  if (lastCommTime + 15000 < now) {
```

Comment:
Can we put the 15000 into a const and maybe reduce the the time to 2x the hearbeat time? 15s seems like a lot!

Reply:
Set it in var to 4sec. 
One will need to raise it up quite a bit, if he need to use breakpoints in the desktop app.